README.md

```diff
@@ -25,7 +25,8 @@ IMPORTANT: The extension must be released using at least Quarkus 1.13.0.Final, o
 
 ### Compatibility with older Quarkus Core versions
 
-Having an extension built against Quarkus Core version `X`, it is assumed that the extension release is compatible with any requested Quarkus version `>= X`. If you want to mark an extension release compatible with an older Quarkus core version, add a `compatible-with-quarkus-core` attribute inside the version, as in the following example:
+Having an extension built against Quarkus Core version X.Y, it is assumed that the extension release is compatible with any requested Quarkus version >= X.Y and less than Z.0. That is, minor Quarkus releases are assumed to be backwards compatible for extensions, but major ones are not.
```

Comment:
Is `Z` missing in the `Quarkus Core version X.Y`?

Reply:
I don't think so, but maybe Z isn't the best choice of letter for Z.0 since it sounds like it wants to go with X and Y. What about `(X+1).0`?

Reply:
Yeah, I think that would explain it better 👍🏻

Reply:
Done and pushed and squashed.